spire-server.yaml

```diff
@@ -53,6 +54,14 @@ subpackages:
           mkdir -p "${{targets.subpkgdir}}"/usr/bin
           install -Dm755 ./bin/spire-agent "${{targets.subpkgdir}}/usr/bin/spire-agent"
 
+  - name: oidc-discovery-provider
```

Comment:
This should have been named spire-oidc-discovery-provider

Reply:
yeah, it should be, let me send a fix for this!

Reply:
#2550